ledger/fullblock_perf_test.go

```diff
@@ -636,3 +639,52 @@ func callAppTransaction(
 	appTx.Type = protocol.ApplicationCallTx
 	return
 }
+
+// BenchmarkBlockEncoding builds a full block of pay transactions and benchmarks the time it takes to encode b.N of them.
+func BenchmarkBlockEncoding(b *testing.B) {
```

Comment:
misnamed? This is really a benchmark of zstd compression at some level? I was expecting a benchmark around the msgp operation as 'encoding'